docs/api/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: API Clients
+layout: page
+section: api
+description: Conjur API clients for integrating our service with your platforms and servers.
+---
+
+We provide APIs for a variety of languages so you deeply integrate Conjur into your software and leverage all its capabilities.
```

Comment:
Tiny grammatical thing: this should be `all of its` with an "of" I believe. I know you just copied this but we might as well fix it now!

Reply:
Never mind, both appear to be correct!